kotsadm/cmd/kotsadm/main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "github.com/replicatedhq/kots/kotsadm/cmd/kotsadm/cli"
+import (
+	"math/rand"
+	"time"
+
+	"github.com/replicatedhq/kots/kotsadm/cmd/kotsadm/cli"
+)
 
 func main() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
why not put the rand.Seed in the gitops init function? https://github.com/replicatedhq/kots/pull/1061/files#diff-861ad608adfe5caaf02438963a6b1c02R16

Reply:
instead of spread through the code

Reply:
idk. typically i feel like it should be left up to the consumer of the package